src/database/index.js

```diff
@@ -0,0 +1,22 @@
+import mongoose from 'mongoose'
+import dotenv from 'dotenv'
+
+dotenv.config()
```

Comment:
Codacy has a fix for the issue: [Insert `;`](https://app.codacy.com/gh/Denniman/group-four-BE/pullRequest?prid=7801432)
```suggestion
dotenv.config();
```